juriscraper/state/texas/common.py

```diff
@@ -0,0 +1,488 @@
+from datetime import datetime
+from enum import Enum
+from typing import TypedDict
+
+from build.lib.juriscraper.lib.html_utils import fix_links_but_keep_anchors
+from lxml import html
+from lxml.html import HtmlElement
+
+from juriscraper.lib.html_utils import clean_html, parse_table
+from juriscraper.lib.string_utils import clean_string, harmonize
+from juriscraper.scraper import Scraper
+
+
+class CourtID(Enum):
+    UNKNOWN = "texas_unknown"
+    SUPREME_COURT = "texas_sc"
```

Comment:
If we use the ID from:

https://search.txcourts.gov/Case.aspx

Shouldn’t the Supreme Court be `texas_cossup`?
https://search.txcourts.gov/Case.aspx?cn=25-1100&coa=cossup


And criminal appeals:
texas_coscca
?

And appellate courts noticed that use a 0 like: coa01, coa02 ... coa09